wordpress-mobile/WordPress-Android
# [Tooling] Update path to PlayStoreStrings.po file for Jetpack

Moves the `metadata/JetpackPlayStoreStrings.po` file to `jetpack_metadata/PlayStoreStrings` to be alongside the other Jetpack metadata files.

### Changes

 - `jetpack_metadata/PlayStoreStrings.po` was previously a symlink to `metadata/JetpackPlayStoreStrings.po`
 - That symlink is now deleted, and replaced by the original file moved to the new location (i.e. I `mv metadata/JetpackPlayStoreStrings.po jetpack_metadata/PlayStoreStrings.po`
 - I updated the Fastfile accordingly to now write content to the `jetpack_metadata/PlayStoreStrings.po` instead of the `metadata/JetpackPlayStoreStrings.po` file

To test:

 - We will need differential D67228-code to be accepted before we can have this take effect.
 - Once it lands, check in GlotPress that the import still works as expected

## Ready for review, but Merging Blocked (pending wpcom Diff approval)

**We will need differential D67228-code to be accepted and merged in before we can merge this.**

_Technically we could merge this beforehand, but if the differential in wpcom ends up taking too long to get approved and merged, and only gets merged after this week, that would mean the Editorial'd release notes for 18.3 wouldn't get imported in time._

So it feels safer to let the Editorial'd release notes for 18.3 land first, and let them get imported as usual, and once we have them imported in GlotPress for this sprint, update this PR accordingly (esp the `.po` file content), land D67228-code and finally merge this.

 - [ ] Get the Editorial'd release notes for 18.3 merged in `release/18.3` then in `develop`
 - [ ] Check that the release notes for 18.3 have been imported in GlotPress
 - [ ] Rebase this PR to include the new release notes in the new file/location (`jetpack_metadata/PlayStoreStrings.po`)
 - [ ] Land D67228-code in wpcom and merge this PR in `release/18.3` then merge `release/18.3` back in `develop` again.
 - [ ] Ensure on the next day that the release notes for 18.3 are still ok in GlotPress (as they would have been imported again but from the new file/location)
